src/Utilities/Resources/Strings.resx

```diff
@@ -172,6 +172,10 @@
   <data name="ToolTask.EnvironmentVariableHeader">
     <value>Environment Variables passed to tool:</value>
   </data>
+  <data name="ToolTask.ValidateParametersFailed">
+    <value>MSB6011: {0}.ValidateParameters() returned false, but did not log an error.</value>
```

Comment:
The user probably did not write the task and so `ValidateParameters()` does not mean anything to them. Would this be more meaningful: `MSB6011: Invalid parameters passed to the {0} task.`

Reply:
Yes, I like that. @chamons, still sound good?